src/Feature/Account/code/Controllers/AccountsController.cs

```diff
@@ -45,7 +45,7 @@ public class AccountsController : BaseController
             ITrackingService trackingService)
         {
             Assert.ArgumentNotNull(accountService, nameof(accountService));
-            Assert.ArgumentNotNull(mapper, nameof(mapper));
+            Assert.ArgumentNotNull(mapper, nameof(IAccountMapper));
```

Comment:
Why there is interface?

Reply:
For more useful massage instead just "mapper"

Reply:
But argument has name `mapper` not `IAccountMapper`